src/telemetry/WorkspaceFileTelemetryProps.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+export interface WorkspaceFileTelemetryProps {
+    dockerfileCount?: string;
```

Comment:
This is new

Reply:
<img src="https://media3.giphy.com/media/3o7bufkPz3LRof205G/giphy.gif"/>

Really need to review this one...